spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/valid/FeignClientTests.java

```diff
@@ -34,9 +34,12 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Objects;
+import java.util.Map;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
We do not allow `com.google` classes.

Reply:
I will update to regular Map. The reason for the test is, I was stuck in a situation where I was using HeaderMap in the FeignClient, which converted the Http method from GET to POST, as it was adding it to the body. Had this test been there, I wouldn't have spent time in researching. Let me know if this helps?